src/app/dashboard/competitions/[competitionId]/register-form/edit/page.tsx

```diff
@@ -0,0 +1,41 @@
+import { notFound } from "next/navigation";
+
+import { getCompetitionById } from "@/lib/db/competition";
+import { getSignedUrlForReading } from "@/lib/s3";
+import { RegisterFormSettings } from "@/types/competition";
+
+import EditCompetitionRegisterForm from "./components/EditCompetitionRegisterForm";
+
+interface EditRegisterFormPageProps {
```

Comment:
Acá lo mismo que antes, le llamaría:


```suggestion
interface EditCompetitionRegisterFormPageProps {
```